src/integration-test/java/com/commercetools/sync/integration/commons/utils/CustomObjectITUtils.java

```diff
@@ -52,6 +56,35 @@ public static void deleteWaitingToBeResolvedCustomObjects(@Nonnull final SphereC
                 .toArray(CompletableFuture[]::new));
     }
 
+    public static void deleteWaitingToBeResolvedTransitionsCustomObjects(@Nonnull final SphereClient ctpClient) {
+
+        final CustomObjectQuery<WaitingToBeResolvedTransitions> customObjectQuery =
+            CustomObjectQuery
+                .of(WaitingToBeResolvedTransitions.class)
+                .byContainer(CUSTOM_OBJECT_CONTAINER_KEY);
```

Comment:
why not (TRANSITIONS_CUSTOM_OBJECT_CONTAINER_KEY) ?

Reply:
630abb4f